src/pages/Home/HomeComponents/VideosPane/VideoCard/VideoCard.jsx

```diff
@@ -4,26 +4,52 @@ import { useState } from "react";
 import { useAuth } from "../../../../../context/auth-context";
 import { useNavigate } from "react-router-dom";
 import { PlaylistModal } from "../../../../../components/PlaylistModal/PlaylistModal";
+import {
+  requestAddVideoInWatchLater,
+  requestDeleteVideoInWatchLater,
+} from "../../../../../utils/server-request";
```

Comment:
You may create a jsconfig.json file in root folder and set base directory as /src
This way you can avoid writing .../... which would result in even cleaner code. You may consider this while finishing up the project.

ref: https://stackoverflow.com/questions/68675994/what-is-jsconfig-json check how to setup baseUrl in compilerOptions

Reply:
thanks for the suggestion tanay, will check more on this